packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -32,12 +32,15 @@ export const SceneTagSettingsEditor: React.FC = () => {
         ...tagSettings,
         ...settingsPartial,
       };
-      const newComponentSettings: IComponentSettingsMap = {
-        ...getSceneProperty(KnownSceneProperty.ComponentSettings),
-        [KnownComponentType.Tag]: newTagSettings,
-      };
+      if (!isViewing) {
+        const newComponentSettings: IComponentSettingsMap = {
+          ...getSceneProperty(KnownSceneProperty.ComponentSettings),
+          [KnownComponentType.Tag]: newTagSettings,
+        };
 
-      setSceneProperty(KnownSceneProperty.ComponentSettings, newComponentSettings);
+        setSceneProperty(KnownSceneProperty.ComponentSettings, newComponentSettings);
```

Comment:
Having the `newComponentSettings` variable wrapped in an `if` check means that this will return `undefined` in the scenario where `isViewing` is true. Is it intentional to pass `undefined` in this scenario?

Reply:
Thanks for confirming.